include/CppUTestExt/MockNamedValue.h

```diff
@@ -42,6 +42,16 @@ class MockNamedValueComparator
     virtual SimpleString valueToString(const void* object)=0;
 };
 
+class MockNamedValueCopier
+{
+public:
+    MockNamedValueCopier() {}
+    virtual ~MockNamedValueCopier() {}
+
+    virtual void copy(const void* in, void* out)=0;
```

Comment:
Please put the output parameter first. Doing so is a common idiom used in many places, SimpleString among others.